### ISSUE TITLE
NOTICE lists Copyright statement with no reference to a LICENSE (Legal Issue : Copyright vs. FOSS)

### ISSUE THREAD

**Issue by bassonya** [hashicorp/terraform-aws-consul#214]
Hello,

I'm working with my legal department and our policies on the use of Free and Open Source Software (FOSS). We are not (yet) interested in any commercial support and acknowledge the limitations around this tool. 

While **terraform-aws-consul** does have a proper Apache 2.0 [LICNESE](https://github.com/hashicorp/terraform-aws-consul/blob/master/LICENSE) the [NOTICE](https://github.com/hashicorp/terraform-aws-consul/blob/master/NOTICE) is causing some questions due to the generic copyright statement and does not direct the reader to anything specific to know where the Apache Permissive license stops and a more restricted copyright picks up.

In my situation I'm working to have [TFLint](https://github.com/terraform-linters/tflint) approved for use which uses the  [MPL-2.0 License model](https://choosealicense.com/licenses/mpl-2.0/). Some of the unit testing code in that project uses this module and the author appropriately referenced the [NOTICE](https://github.com/hashicorp/terraform-aws-consul/blob/master/NOTICE). Here's where the issue comes into play.

If the [NOTICE](https://github.com/hashicorp/terraform-aws-consul/blob/master/NOTICE) asserts a general Copyright statement and there is no reference to the Apache 2.0 [LICNESE](https://github.com/hashicorp/terraform-aws-consul/blob/master/LICENSE) in that notice then the legal experts are claiming that the bounds of the Apache 2.0 license vs. what is actually copyrighted is too vague.

In short, for the sake of clarity I'm asking that the [NOTICE](https://github.com/hashicorp/terraform-aws-consul/blob/master/NOTICE) reference the Apache 2.0 [LICNESE](https://github.com/hashicorp/terraform-aws-consul/blob/master/LICENSE) so that the "[copyleft licensing](https://en.wikipedia.org/wiki/Copyleft)" is crystal clear for this and related projects.

**Comment by brikis98** [comment]
Great question! The intention is definitely for all the code to be under the Apache 2.0 license. Let me check with our legal team on the right thing to do in the NOTICE and get back to you!

**Comment by brikis98** [comment]
OK, heard back from our legal team. Sounds like calling out the license in the `NOTICE` file is not a strict requirement—the way the repo is laid out is clear enough—but there's no harm in adding it either. See https://github.com/hashicorp/terraform-aws-consul/pull/215.
